modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/channel-advection-slip.i

```diff
@@ -0,0 +1,321 @@
+    v_slip = 'vel_slip_y'
+    velocity_interp_method = ${velocity_interp_method}
+    advected_interp_method = 'upwind'
+  []
+  [phase_2_src]
+    type = NSFVInterfaceTransfer
+    variable = phase_2
+    phase_coupled = phase_1
+    alpha = 0.1
+  []
+[]
+
+[FVBCs]
+  [inlet-u]
+    type = INSFVInletVelocityBC
+    boundary = 'left'
+    variable = vel_x
+    functor = '${U}'
+  []
+  [inlet-v]
+    type = INSFVInletVelocityBC
+    boundary = 'left'
+    variable = vel_y
+    functor = '0'
+  []
+  [walls-u]
+    type = INSFVNoSlipWallBC
+    boundary = 'top bottom'
+    variable = vel_x
+    function = 0
+  []
+  [walls-v]
+    type = INSFVNoSlipWallBC
+    boundary = 'top bottom'
+    variable = vel_y
+    function = 0
+  []
+  [outlet_p]
+    type = INSFVOutletPressureBC
+    boundary = 'right'
+    variable = pressure
+    function = '0'
+  []
+  [inlet_phase_1]
+    type = FVDirichletBC
+    boundary = 'left'
+    variable = phase_1
+    value = '${fparse 1.0 - inlet_phase_2}'
+  []
+  [inlet_phase_2]
+    type = FVDirichletBC
+    boundary = 'left'
+    variable = phase_2
+    value = ${inlet_phase_2}
+  []
+[]
+
+[AuxVariables]
+  [vel_slip_x]
+    type = MooseVariableFVReal
+  []
+  [vel_slip_y]
+    type = MooseVariableFVReal
+  []
+  [drag_coefficient]
+    type = MooseVariableFVReal
+  []
+  [rho_mixture_var]
+    type = MooseVariableFVReal
+  []
+  [mu_mixture_var]
+    type = MooseVariableFVReal
+  []
+  [phase_1]
+    type = MooseVariableFVReal
+  []
+[]
+
+[AuxKernels]
+  [populate_u_slip]
+    type = INSFVSlipVelocityAux
+    variable = 'vel_slip_x'
+    momentum_component = 'x'
+    u = 'vel_x'
+    v = 'vel_y'
+    rho = ${rho}
+    mu = 'mu_mixture'
+    rho_d = ${rho_d}
+    particle_diameter = ${dp}
+    linear_coef_name = 'Darcy_coefficient'
+  []
+  [populate_v_slip]
+    type = INSFVSlipVelocityAux
+    variable = 'vel_slip_y'
+    momentum_component = 'y'
+    u = 'vel_x'
+    v = 'vel_y'
+    rho = ${rho}
+    mu = 'mu_mixture'
+    rho_d = ${rho_d}
+    particle_diameter = ${dp}
+    linear_coef_name = 'Darcy_coefficient'
+  []
+  [populate_cd]
+    type = FunctorAux
+    variable = drag_coefficient
+    functor = 'Darcy_coefficient'
+  []
+  [populate_rho_mixture_var]
+    type = FunctorAux
+    variable = rho_mixture_var
+    functor = 'rho_mixture'
+  []
+  [populate_mu_mixture_var]
+    type = FunctorAux
+    variable = mu_mixture_var
+    functor = 'mu_mixture'
+  []
+  [compute_phase_1]
+    type = ParsedAux
+    variable = phase_1
+    coupled_variables = 'phase_2'
+    expression = '1 - phase_2'
+  []
+[]
+
+[Materials]
+  [CD]
+    type = NSFVDispersePhaseDragMaterial
+    rho = 'rho_mixture'
+    mu = mu_mixture
+    u = 'vel_x'
+    v = 'vel_y'
+    particle_diameter = ${dp}
+  []
+  [mixing_material]
+    type = NSFVMixtureMaterial
+    phase_2_names = '${rho} ${mu}'
+    phase_1_names = '${rho_d} ${mu_d}'
+    prop_names = 'rho_mixture mu_mixture'
+    phase_1_fraction = 'phase_2'
+  []
```

Comment:
this is re-implementing the mixture fluid properties in the fluid properties module.

now I know it s not crazy complicated but we should make sure both are usable.
I think it might just a material of implementing a GeneralFunctorMixtureFluidProps object ?

@joshuahansel @lindsayad have you worked with the mixture FPs?

Reply:
What's in the FP module is `IdealRealGasMixtureFluidProperties`, which mixes various gases. It is not just doing a volume-fraction-weighted sum.

Reply:
what is it doing?

Reply:
It's not well documented (big surprise). It's doing a gas mixture model. The mixture pressure is the sum of the partial pressures. Some quantities do work out to be weighted averages. Some don't.

Reply:
This general material is also used for solidification. It just taking the weighted average of two materials, they can be gases, fluids, or solids. For a mixture model, you expect the thermophysical properties to be mixed. See for example, the [COMSOL](https://doc.comsol.com/6.2/docserver/#!/com.comsol.help.cfd/cfd_ug_fluidflow_multi.09.148.html?highlight=mixture+mixture%25E2%2590%25A4model+model) documentation

Reply:
The stuff in the FP module is more advanced but also too specialized.
I think we should rework this later.

in the FP module we should move from "VaporMixtureFP" to "MixtureFP" (or some other name), then move NS uses to classes that derive from MixtureFP.

Do we only mix 2 quantities for your uses @tanoret ?
in the FP module, the vapors can be arbitrarily many.

Reply:
You say "mixture" for multiphase, when the FP module was differentiating between mixtures and multi-phase.

Maybe the vaporMixture is not the right class to look at in FP, but rather the two phase fluid properties classes

Reply:
> Do we only mix 2 quantities for your uses @tanoret ?

Yes, one "effective" liquid and one "effective" gas/vapor. You can reuse other functors to compute the "effective" properties properties of the liquid and gas. However, in 99% of cases, you will have a single species liquid and a single species gas